client/extensions/woocommerce/app/order/order-details/row-refund.js

```diff
@@ -17,11 +18,11 @@ class OrderRefundRow extends Component {
 			refunds: PropTypes.array.isRequired,
 		} ),
 		showTax: PropTypes.bool,
-	}
+	};
 
-	getRefundedTotal = ( order ) => {
-		return order.refunds.reduce( ( total, i ) => total + ( i.total * 1 ), 0 );
-	}
+	getRefundedTotal = order => {
+		return order.refunds.reduce( ( total, i ) => total + i.total * 1, 0 );
```

Comment:
Yeah, though that's less clear now that prettier's removed the parens.

Reply:
Yep kind of what I thought too, maybe we can either just change this one line back - or use `parseFloat` or something

Reply:
prettier runs automatically on pre-commit, so even if I changed it back it would be stripped. Using parseFloat would be clearer, too -- i'll make that change & merge this 👍